backup.sh

```diff
@@ -1,10 +1,18 @@
 #!/bin/bash
 
+# Get hostname: try read from file, else get from env
+[ -z "${MYSQL_HOST_FILE}" ] || { MYSQL_USER=$(head -1 "${MYSQL_HOST_FILE}"); }
```

Comment:
I think you want to extract the hostname from the file, not the MYSQL_USER. Check this assignment